acs-integration-tests/src/test/java/com/ge/predix/acs/cloudfoundry/PushAcsApplication.java

```diff
@@ -24,7 +24,7 @@
 @Component
 final class PushAcsApplication {
 
-    @Value("${ACS_SERVICE_ARTIFACT_DIR:../service/target}")
+    @Value("${ACS_SERVICE_ARTIFACT_DIR:../../acs/service/target}")
```

Comment:
This isn't necessary since the current working directory is `acs/acs-integration-tests` and thus `../service/target` and `../../acs/service/target` resolve to the same absolute path